extensions/tmtv/src/commandsModule.js

```diff
@@ -553,7 +553,10 @@ const commandsModule = ({
         commandsManager.runCommand('setViewportColormap', {
           viewportIndex,
           displaySetInstanceUID: ptDisplaySet.displaySetInstanceUID,
-          colormap,
+          colormap: {
```

Comment:
This line is the entire change - the colormap is a structure, not a simple object, so it needs to be set correctly.